packages/data-store/src/entities/presentationDefinition/PresentationDefinitionItemEntity.ts

```diff
@@ -0,0 +1,40 @@
+import { BaseEntity, BeforeInsert, BeforeUpdate, Column, CreateDateColumn, Entity, Index, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm'
+import { IsNotEmpty } from 'class-validator'
+
+@Entity('PresentationDefinitionItem')
+@Index(['version'], { unique: false })
+export class PresentationDefinitionItemEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @Column({ name: 'definition_id', length: 255, nullable: false, unique: false })
+  @IsNotEmpty({ message: 'definitionId field must not be empty' })
```

Comment:
I don't understand the question, the entity has nullable: false

Reply:
You can. All he is saying the error message it too technical

Reply:
fixed in next commit

Reply:
not fixed

Reply:
What's not fixed then?
```
@IsNotEmpty({ message: 'A blank definitionId field is not allowed' })
```
This is how all @IsNotEmpty messages are. 
Please post what the message should be, otherwise I'd need do a few more trial & errors on this.